client/ratings_reviews/helpers.js

```diff
@@ -148,7 +151,16 @@ const formatReviewTile = (summary, body, photos) => {
   if (photos.length !== 0) {
     showPhotos = true;
   }
-  return [summary, body, additionalBody, showAdditionalBodyButton, showPhotos];
+  var currentHelpfulReviews = JSON.parse(sessionStorage.getItem('helpfulReviews'));
+  if (currentHelpfulReviews.indexOf(reviewId) !== -1) {
+    helpful = 1;
+    showAddHelpfulButton = false;
+  }
+  var currentReportedReviews = JSON.parse(sessionStorage.getItem('reportedReviews'));
+  if (currentReportedReviews.indexOf(reviewId) !== -1) {
+    reportStatus = true;
+  }
+  return [summary, body, additionalBody, showAdditionalBodyButton, showPhotos, helpful, showAddHelpfulButton, reportStatus];
```

Comment:
Could this be an object literal instead, or does it have to be an array for some reason? Not sure which is the best approach, totally depends on how you're accessing the data in other areas of your code, just a thought I had!

Reply:
Hi @jabrake, this is an extremely valid point! When I first wrote this function, it was just for a couple of features so using an array wasn't that bad. I think we're at the point where an object would be more appropriate. I will make notes of this and refactor in later PR. Thank you!